angular-hub/src/app/pages/index.page.ts

```diff
@@ -76,6 +74,17 @@ export const routeMeta: RouteMeta = {
         <li>
           <app-event-card [event]="event"></app-event-card>
         </li>
+      } @empty {
+        <p class="col-span-2">
+          No event matching the filter
+          @if (date()) {
+            {{ date() | date: 'MM/DD/YYYY' }}
```

Comment:
What do you think about formats like:
1. `YYYY-MM-DD` (ISO)
1. `dd MMM YYYY` (dd -> 1 instead of "01", and MMM -> "May" instead of 05)

2 is my preferred way to represent dates to make it readable while avoiding debates on American/ISO date format. In my country, we write dates in the `DD/MM/YYYY` format, so I would prefer something neutral 😅

Reply:
Hmm ok, open for discussion and set whatever format is the most inclusive. @geromegrignon wdyt?